lib/matplotlib/figure.py

```diff
@@ -2076,6 +2081,9 @@ def get_constrained_layout_pads(self, relative=False):
         """
         return self._parent.get_constrained_layout_pads(relative=relative)
 
+    def get_layout_engine(self):
```

Comment:
If this is never called, do we need a test of adding a colorbar in a sub-figure?

Reply:
It does get called by `test_figure.py::test_subfigure`   - I just tested via print statements (my highly professional debugging method).  I don't understand why codecov doesn't think it is being hit.

Reply:
I think this was #22178 and codecov not having been updated for the branch rename.